### ISSUE TITLE
Warn users when they try and build with Java 1.9, which fails.

### ISSUE THREAD

**Issue by AWoloszyn** [google/gapid#1734]
Just so that users can have a better time trying to resolve build issues.

**Comment by ben-clayton** [comment]
I thought this was fixed with https://github.com/google/gapid/pull/1503. Pascal?

**Comment by pmuetschard** [comment]
No, #1503 fixed running gapid in Java 9, but this is about building with Java 9, where compiling protobuf fails.
